scripts/update-protos.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+# This script is used to update the vendored proto files from their source repositories
+#
+# Note: This currently just copies the proto files from the upstream containerd projects
+# into their respective vendored directories. There may be additional work involved to
+# update the respective crates (manually) after running the script.
+
+DIR=$(dirname "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )")
+TEMP_DIR=`mktemp -d -p "$DIR"`
+
+CONTAINERD_DIR="${TEMP_DIR}/containerd"
+git clone https://github.com/containerd/containerd --depth 1 $CONTAINERD_DIR
```

Comment:
Could you pls add `git clone --branch $CONTAINERD_BRANCH`, so we can control which protos we're getting here.

> -b <name>, --branch <name>
> Instead of pointing the newly created HEAD to the branch pointed to by the cloned repository’s HEAD, point to <name> branch instead. In a non-bare repository, this is the branch that will be checked out.  --branch can also take tags and detaches the HEAD at that commit in the resulting repository.